docs/website/versioned_docs/version-3.0.0/command-reference/logs.md

```diff
@@ -19,4 +19,10 @@ init` command.
 
 Note that if multiple containers are named the same (for example, `main`), the `odo logs` output appends a number to 
 container name to help differentiate between the containers. In the output, you will see containers named as `main`, 
-`main[1]`, `main[2]`, so on and so forth.
\ No newline at end of file
+`main[1]`, `main[2]`, so on and so forth.
+
+It also supports `--follow` flag which allows you to follow/tail/stream the logs of the containers. It works by using 
+the same commands as above albeit, with a `--follow` flag:
+* Use `odo logs --dev` to follow the logs for the containers created by `odo dev` command.
```

Comment:
Could you add the `--follow` flag to the commands?

Reply:
Oops. 😆